lib/oracle.js

```diff
@@ -36,6 +85,14 @@ export class OracleSingleton {
       );
     return OracleSingleton.instance;
   }
+
+  static getTypes() {
```

Comment:
Since this method doesn't actually do anything, I would consider making it a getter function simply called "types".